.gitignore

```diff
@@ -25,3 +25,4 @@ __snapshots__
 
 # debug & other vs code settings
 .vscode
+server/models/Request.js
```

Comment:
This isn't a good idea, if we commit this change any further modifications to the `Request` model will be untracked. You can add this to your local `.gitignore` temporarily but ideally we don't have this pushed to the repo.